de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/core/ui/util/CollaborationUtils.java

```diff
@@ -267,8 +266,8 @@ private static String getShareProjectDescription(
                     List<IResource> resources = sarosSession
                         .getSharedResources(project);
 
-                    fileCountAndSize = FileUtils
-                        .getFileCountAndSize(resources, false, IResource.NONE);
+                    fileCountAndSize = getFileCountAndSize(resources, false,
+                        IResource.NONE);
 
                     result.append(String
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: %n should be used in place of \n to produce the platform-specific line separator. ([squid:S3457](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3457))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=308&issues=AWcYCgrzkEeD8QRk2pY3&open=AWcYCgrzkEeD8QRk2pY3)

[ISSUE_KEY]: AWcYCgrzkEeD8QRk2pY3